l10n_co_geo/__init__.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+###############################################################################
+#    License, author and contributors information in:                         #
+#    __openerp__.py file at the root folder of this module.                   #
+###############################################################################
+
+from openerp import SUPERUSER_ID
+from models import (
+        res_country_state_city,
+        res_country_state,
+        res_country,
+        res_partner,
+        res_company
+)
+
+
+def _fill_city_id(cr, registry):
+    ''' This is a utiliy function to help fill res_partner city_id, state and country with Bogota, Dc, Colombia.
```

Comment:
Good idea! Two questions: 1. Is this OCA standard or even Odoo standard? 2. Only for init hooks or is the idea to have any kind of hook, also in normal code, clustered in a special file. ... I definitely see the use for this practice.